charts/llm-engine/templates/_helpers.tpl

```diff
@@ -330,7 +330,7 @@ volumeMounts:
     mountPath: /workspace/llm_engine/llm_engine/infra/gateways/resources/templates
   {{- if .Values.aws }}
   - name: config-volume
-    mountPath: /root/.aws/config
+    mountPath: /home/user/.aws/config
```

Comment:
iirc this means `~ = /home/user` instead of `/root` on the containers? I guess the change was made for a reason